llm/server.go

```diff
@@ -505,11 +502,11 @@ func (s *llmServer) getServerStatus(ctx context.Context) (ServerStatus, error) {
 	}
 
 	switch status.Status {
-	case "ok":
+	case ServerStatusReady.ToString():
```

Comment:
Out of scope since you didn't really change this:

1. avoid comparing strings where possible. This can probably use `errors.As`
2. Status should implement [`fmt.Stringer`](https://pkg.go.dev/fmt#Stringer)

Reply:
I made this change, may as well keep it tidy while I'm in the neighborhood